src/uk/gov/hmcts/contino/GradleBuilder.groovy

```diff
@@ -256,6 +256,13 @@ EOF
     return gradleWithOutput("buildEnvironment").contains(pluginName)
   }
 
+  @Override
+  def securityScan() {
+    localSteps.writeFile(file: 'security.sh', text: localSteps.libraryResource('uk/gov/hmcts/pipeline/security/security-backend/security.sh'))
+    this.securitytest.execute()
```

Comment:
you need to create the `SecurityScan` class in the constructor so that this works

something like:
`this.securityScan = new SecurityScan(steps)`